test_scripts/API/Restructuring_OnResetTimeout/001_Success_floow.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 3000) to SDL
+-- 4) HMI sends response in 12 seconds after receiving request
+-- SDL does:
+-- 1) Receive response and successful process it
+-- 2) Wait response from HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("App registration", common.registerAppWOPTU)
+runner.Step("App activation", common.activateApp)
+runner.Step("Create InteractionChoiceSet", common.createInteractionChoiceSet)
+
+runner.Title("Test")
+runner.Step("Send SendLocation", common.sendLocation, { "SendLocation", 3000, 12000 })
+runner.Step("Send Alert", common.alert, { "Alert", 3000, 12000 })
```

Comment:
@HSavynetska in this case: new timeout will be =  3000 (pResetPeriod) and value of pWait should be less that pResetPeriod.
mob -> SDL: SendLocation 
SDL->HMI: SendLocation 
--- SDL starts default timeout = 10sec
HMI->SDL: OnResetTimeout {resetPeriod = 3000} --- new timeout will be **3sec**
so for successful RPC processing SDL should receive a response from HMI <= than 3 seconds.
Or you can increase timeout value.
For example 
runner.Step("Send SendLocation", common.sendLocation, { "SendLocation", 13000, 12000 })
new timeout 13sec, wait 12sec and then send a response from HMI and expect it on mob side

Reply:
@LitvinenkoIra update